master/internal/core.go

```diff
@@ -365,7 +377,12 @@ func (m *Master) Run() error {
 
 	proxyRef, _ := m.system.ActorOf(actor.Addr("proxy"), &proxy.Proxy{})
 
-	m.initializeResourceProviders(proxyRef, provisionerSlotsPerInstance)
+	cert, err := m.readTLSCertificate()
+	if err != nil {
+		return err
```

Comment:
want to `errors.Wrap` this too? In my short time here, i've seen a few completely unwrapped errors and they are just a total mystery.

Reply:
Yep, done.